packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -460,4 +470,16 @@ export class WorkflowsController {
 			workflow,
 		});
 	}
+
+	@Put('/:workflowId/transfer')
+	@Licensed('feat:advancedPermissions')
```

Comment:
No sure about this, we've put lots of other endpoints behind the admin feature flag.

Reply:
It was decided that this endpoint is not behind a license flag.